app/uk/gov/hmrc/tariffclassificationfrontend/controllers/SuspendCaseController.scala

```diff
@@ -35,18 +37,31 @@ class SuspendCaseController @Inject()(authenticatedAction: AuthenticatedAction,
                                       val messagesApi: MessagesApi,
                                       implicit val appConfig: AppConfig) extends RenderCaseAction {
 
+  private val form: Form[Boolean] = MandatoryBooleanForm.form("suspend_case")
+
   override protected val config: AppConfig = appConfig
   override protected val caseService: CasesService = casesService
 
   override protected def redirect: String => Call = routes.CaseController.applicationDetails
   override protected def isValidCase(c: Case)(implicit request: AuthenticatedRequest[_]): Boolean = c.status == OPEN
 
   def suspendCase(reference: String): Action[AnyContent] = authenticatedAction.async { implicit request =>
-    getCaseAndRenderView(reference, c => successful(views.html.suspend_case(c)))
+    getCaseAndRenderView(reference, c => successful(views.html.suspend_case(c, form)))
   }
 
   def confirmSuspendCase(reference: String): Action[AnyContent] = authenticatedAction.async { implicit request =>
-    getCaseAndRenderView(reference, casesService.suspendCase(_, request.operator).map(views.html.confirm_with_contact_details(_, "suspended")))
+    form.bindFromRequest().fold(
+      errors => {
+        getCaseAndRenderView(
```

Comment:
line 49 and line 55 can reuse a common method

Reply:
As discussed I'm going to leave this as it's already readable